src/Server_Command.php

```diff
@@ -80,11 +80,12 @@ function __invoke( $_, $assoc_args ) {
 
 		// Get the path to the router file
 		$command_root = WP_CLI\Utils\phar_safe_path( dirname( __DIR__ ) );
-		$router_path = $command_root . '/router.php';
+		$router_path  = $command_root . '/router.php';
 		if ( ! file_exists( $router_path ) ) {
 			WP_CLI::error( "Couldn't find router.php" );
 		}
-		$cmd = \WP_CLI\Utils\esc_cmd( '%s -S %s -t %s -c %s %s',
+		$cmd = \WP_CLI\Utils\esc_cmd(
```

Comment:
Use an import for `WP_CLI\Utils`:
```suggestion
		$cmd = Utils\esc_cmd(
```